packages/fiber/src/core/stages.ts

```diff
@@ -1,17 +1,8 @@
-import { MutableRefObject } from 'react'
 import { StoreApi, UseBoundStore } from 'zustand'
-import { RootState } from './store'
+import { RootState, Subscription } from './store'
 
-export interface UpdateCallback {
-  (state: RootState, delta: number, frame?: XRFrame): void
-}
-
-export type UpdateCallbackRef = MutableRefObject<UpdateCallback>
-type Store = UseBoundStore<RootState, StoreApi<RootState>>
-export type UpdateSubscription = { ref: UpdateCallbackRef; store: Store }
-
-export type FixedStageOptions = { fixedStep?: number; maxSubsteps?: number }
-export type FixedStageProps = { fixedStep: number; maxSubsteps: number; accumulator: number; alpha: number }
+// TODO: Remove deprecated fields in `Subscription`
+export type UpdateSubscription = Omit<Subscription, 'priority'>
```

Comment:
I didn't see use of `FixedStageOptions` or `FixedStageProps` anywhere, but what's going on with `UpdateCallback` and `RenderCallback`? Should we alias or rename/depreciate one?